ci/playbooks/tcib/run.yml

```diff
@@ -34,40 +34,29 @@
           {%- endif %}
           -e "@{{ ansible_user_dir }}/ci-framework-data/artifacts/parameters/zuul-params.yml"
 
-    - name: Slurp dlrn md5 hash
-      become: true
-      vars:
-        _repo_path: >-
-          {%- if cifmw_repo_setup_output is defined %}
-          {{ cifmw_repo_setup_output }}
-          {%- else %}
-          {{ ansible_user_dir }}/ci-framework-data/artifacts/repositories
-          {%- endif %}
-      block:
-        - name: Check DLRN md5 exists or not
-          ansible.builtin.stat:
-            path: "{{ _repo_path }}/delorean.repo.md5"
-          register: _check_dlrn_md5
-
-        - name: Slurp dlrn md5 hash
-          ansible.builtin.slurp:
-            path: "{{ _repo_path }}/delorean.repo.md5"
-          register: _md5_data
-          when: _check_dlrn_md5.stat.exists
+    - name: Include inner ansible vars file
+      ansible.builtin.slurp:
+        src: "{{ cifmw_artifacts_basedir }}/artifacts/ansible-vars.yml"
+      register: inner_ansible
 
-        - name: Store md5 content
-          ansible.builtin.set_fact:
-            _dlrn_md5: "{{ _md5_data['content'] | b64decode }}"
-          when: _check_dlrn_md5.stat.exists
+    - name: Get inner ansible vars
+      ansible.builtin.set_fact:
+        inner_ansible_vars: "{{ inner_ansible.content | b64decode | from_yaml }}"
```

Comment:
Can we use a block to avoid decoding the same var 3 times and to avoid the fact?

Reply:
Done